dev_guide/deployments/basic_deployment_operations.adoc

```diff
@@ -0,0 +1,469 @@
+`ConfigChange` trigger and `automatic=false`, or with `automatic=true`) and the
+`ImageStreamTag` pointed by the `ImageChange` trigger does not exist yet, then
+the initial deployment process will automatically start as soon as an image is
+imported or pushed by a build to the `ImageStreamTag`.
+====
+
+[[deployment-triggers-using-the-command-line]]
+==== Using the Command Line
+
+The `oc set triggers` command can be used to set a deployment trigger for a
+deployment configuration. For the example above, you can set the
+`ImageChangeTrigger` by using the following command:
+
+----
+$ oc set triggers dc/frontend --from-image=myproject/origin-ruby-sample:latest -c helloworld
+----
+
+For more information, see:
+
+----
+$ oc set triggers --help
+----
+
+
+
+[[deployment-resources]]
+== Setting Deployment Resources
+
+A deployment is completed by a pod that consumes resources (memory and CPU) on a
+node. By default, pods consume unbounded node resources. However, if a project
+specifies default container limits, then pods consume resources up to those
+limits.
+
+You can also limit resource use by specifying resource limits as part of the
+deployment strategy. Deployment resources can be used with the Recreate,
+Rolling, or Custom deployment strategies.
+
+In the following example, each of `resources`, `cpu`, and `memory` is
+optional:
+
+====
+[source,yaml]
+----
+type: "Recreate"
+resources:
+  limits:
+    cpu: "100m" <1>
+    memory: "256Mi" <2>
+----
+
+<1> `cpu` is in CPU units: `100m` represents 0.1 CPU units (100 * 1e-3).
+<2> `memory` is in bytes: `256Mi` represents 268435456 bytes (256 * 2 ^ 20).
+====
+
+However, if a quota has been defined for your project, one of the following two
+items is required:
+
+- A `resources` section set with an explicit `requests`:
++
+====
+[source,yaml]
+----
+  type: "Recreate"
+  resources:
+    requests: <1>
+      cpu: "100m"
+      memory: "256Mi"
+----
+<1> The `requests` object contains the list of resources that correspond to
+the list of resources in the quota.
+====
+
+ifdef::openshift-enterprise,openshift-dedicated,openshift-origin[]
+- A xref:../admin_guide/limits.adoc#admin-guide-limits[limit range] defined in your project, where the
+defaults from the `LimitRange` object apply to pods created during the
+deployment process.
+endif::[]
+ifdef::openshift-online[]
+- A limit range defined in your project, where the
+defaults from the `LimitRange` object apply to pods created during the
+deployment process.
+endif::[]
+
+Otherwise, deploy pod creation will fail, citing a failure to satisfy quota.
+
+[[scaling]]
+== Manual Scaling
+
+In addition to rollbacks, you can exercise fine-grained control over
+the number of replicas from the web console, or by using the `oc scale` command.
+For example, the following command sets the replicas in the deployment
+configuration `frontend` to 3.
+
+----
+$ oc scale dc frontend --replicas=3
```

Comment:
Can you also add an example of setting up an autoscaler? `oc autoscale`. Follow-up to this PR is fine.